krux_marathon_api/marathonapi.py

```diff
@@ -1,63 +1,44 @@
 #!/usr/bin/env python
 
-import krux.logging
+#
+# Standard libraries
+#
+
+from __future__ import absolute_import
 import sys
 import socket
 import json
-from pprint import pprint
+
+#
+# Third party libraries
+#
+
 from marathon import MarathonClient
 from marathon.models import MarathonApp
+
+#
+# Internal libraries
+#
+
+import krux.logging
 from krux.cli import Application, get_group, get_parser
 
 
-class MarathonClientApp(Application):
+class KruxMarathonClient(object):
+    #VERSION = 0.1
 
     def __init__(self):
-        super(MarathonClientApp, self).__init__(name='marathonclientapp', syslog_facility='local0')
-        self.marathon_host = self.args.host
-        self.marathon_port = self.args.port
-        self.marathon_list_apps = self.args.list_apps
-        self.marathon_config_file = self.args.config_file
-
-    def add_cli_arguments(self, parser):
-        super(MarathonClientApp, self).add_cli_arguments(parser)
-        group = get_group(parser, self.name)
-
-        group.add_argument(
-            '--host',
-            default='localhost',
-            help='Marthon server host name or IP address. ',
-        )
-        group.add_argument(
-            '--port',
-            default='8080',
-            help='Marthon server port. ',
-        )
-        group.add_argument(
-            '--list_apps',
-            action='store_true',
-            default=False,
-            help='Use flag to list all marathon apps. ',
-        )
-        group.add_argument(
-            '--config_file',
-            help='Marathon config file to configure app. ',
-        )
-        group.add_argument(
-            '--app',
-            default=False,
-            help='app flag can be used to pass an app name to get '
-            'more detailed info on a running app ',
-        )
-        group.add_argument(
-            '--delete',
-            action='store_true',
-            default=False,
-            help='Set flag to delete an app '
-        )
+
+        NAME = 'KruxMarathonClient'
+        self.logger = krux.logging.get_logger(name=NAME)
 
     def connect(self, address, port):
-        self.logger.info("Testing connection to marthon server")
+        """
+        Tests connecting to the Marathon Server given. Takes in a host and port.
+        If the values weren't passed on the command line, defaults will be used.
+        If the connection fails, the program will stop and exit with a value of 1.
+        """
+        self.logger.info("Testing connection to marathon server")
```

Comment:
Nit: Doing this in one `info()` is probably better. That way you won't get the time stamp on each of them. If you prefer to have a line break in your log, you can use `\n` or `"""Text"""` format.